Components/Image/index.css

```diff
@@ -0,0 +1,51 @@
+.kash-img-fluid {
+  width: 100%;
+  /* height: 300px; */
```

Comment:
Remove comments which are not needed

Reply:
ya thaught of removing all comments at once